### PR TITLE
Upgrade to v0.13

### PR DESCRIPTION
**Pull request by tim-rix**
#### Developer Checklist

- [X] The README contains the correct list of dependencies, inputs, and outputs (e.g., `terraform-docs markdown .` has been run)

#### What does this PR do?

Updates the module to terraform version 0.13

- Removes Travis CI (travis.yml, Makefile) integration
- Adds PR template
- Adds a Github Actions workflow for Tf validation
- Updates module references to 0.13
- Adds required_providers block to versions.tf
- Adds appropriate type statements where they are missing in variables.tf

#### Helpful background context

I tested this module by embedding it locally in the libraries-assets repo that is also being upgraded to Terraform v0.13

#### What are the relevant tickets?

[IN-231](https://mitlibraries.atlassian.net/browse/IN-231)

#### Requires Database Migrations?
 NO

#### Includes new or updated dependencies?
 NO

### CODE REVIEW

**Comment by vab** [comment]
What is this for: variable "null"?

It doesn't look like it's used in the repository.

**Comment by tim-rix** [comment]
I removed the unused "null" variable and the README reference to it.

File: variables.tf [hunk 1]
```diff
@@ -43,72 +43,86 @@ variable "use_regional_s3_endpoint" {
...
 }
 
 variable "compress" {
+  type        = bool
   default     = "false"
   description = "Compress content for web requests that include Accept-Encoding: gzip in the request header"
 }
 
 variable "is_ipv6_enabled" {
+  type        = bool
```
Comment (zotoMIT) [inline variables.tf:77]: If these are type "bool" then the default needs to be unquoted.

**Review by zotoMIT — APPROVED** [review]
Other than the type issue, lgtm

File: variables.tf (hunk 1 above)
Comment (tim-rix) [reply variables.tf:77]: Unquoted type "bool"

**Review by vab — APPROVED** [review]

**Pull request merged**